### PR TITLE
styled accordion 

### PR DESCRIPTION
**Pull request by Dannyboy222**
I styled accordion. not able to push the code because this file is the inside the node_module folder -> react-accessible-accordion -> dist -> #fancy-example (css file)

```css


.accordion {
    border: 1px solid #0000001a;
    border-radius: 2px;
}

.accordion__item + .accordion__item {
    border-top: 2px solid rgba(0, 0, 0, 0.1);
    width: 600px;
}

.accordion__button {
    background-color: #dcd6f7;
    color: #444;
    cursor: pointer;
    padding: 18px;
    width: 100%;
    border: none;
}

.accordion__button:hover {
    background-color: #a6b1e1;
    cursor: pointer;
}

.accordion__button:before {
    position: absolute;
    content: '+';
    height: 25px;
    width: 25px;
    margin-right: 12px;
    right: 13%;
}

.accordion__button[aria-expanded='true']::before,
.accordion__button[aria-selected='true']::before {
    position: absolute;
    content: '-';
    height: 25px;
    width: 25px;
    margin-right: 12px;
    font-size: 25px;

}

[hidden] {
    display: none;
}

.accordion__panel {
    padding: 20px;
    animation: fadein 0.35s ease-in;
}

/* -------------------------------------------------- */
/* ---------------- Animation part ------------------ */
/* -------------------------------------------------- */

@keyframes fadein {
    0% {
        opacity: 0;
    }

    100% {
        opacity: 1;
    }
}

@media only screen and (max-width: 767px) {
    .accordion {
        width: 100%
    }
    
   
};



```

### CODE REVIEW

**Comment by shiv-chan** [comment]
@Dannyboy222 Thanks for your try, @Dannyboy222, however, the style was collapsed and a bit different from our mock-up design. (I liked the colour changes when it's hovered tho ✨ )

I fixed the layout and the styling.
Also, I applied the props to let all accordion items close or open.
You can learn how to do it by reading [the official documentation](https://www.npmjs.com/package/react-accessible-accordion) or [demo](https://react-accessible-accordion.springload.co.nz/).
These links are the ones that I have sent you several times before to give you a hint.
Please read them closely next time or ask us what you can't get or what you are struggling with rather than holding it yourself 🙏 

@miyabitanimchi @Dannyboy222 
As you see, I created the new CSS file called `accordion.css` so that this styling will be applied when it's deployed.

Thanks!

File: client/src/common/components/Home.tsx [hunk 1]
```diff
@@ -81,10 +80,16 @@ const Home = () => {
...
-				<h2 className="text-2xl font-bold text-center md:text-5xl">Q&As</h2>
-				<Accordion className="p-10 text-sm md:text-2xl w-auto">
-					<AccordionItem className="p-5">
+			<section className="p-10 md:p-20 relative">
+				<h2 className="text-2xl md:text-5xl text-center font-bold mb-10 md:mb-20">
+					Q&As
+				</h2>
+				<Accordion
+					allowMultipleExpanded
+					allowZeroExpanded
```
Comment (shiv-chan) [inline client/src/common/components/Home.tsx:89]: @Dannyboy222 these props are the ones you wanted to add.

**Review by shiv-chan — APPROVED** [review]
I updated the code.

@Dannyboy222 @miyabitanimchi Please review the latest one.

Thanks.

File: client/src/common/components/Home.tsx (hunk 1 above)
Comment (Dannyboy222) [reply client/src/common/components/Home.tsx:89]: thank you!

**Review by miyabitanimchi — APPROVED** [review]
Looks perfect ✨ Thank you for your hard work @Dannyboy222 and @shiv-chan 🙌

**Pull request merged**